disparate-flies/WorkforceManagement
# Am comp model

Please structure your pull requests this way:

**Purpose of PR** : add Computer Model in order to access information about computers

**Issue number this request impacts** : #23 

**Description of what was changed:**  created the get and set for purchasedate, decommissiondate, model, and condition

 **Thorough list of steps to test**: 

1. cd into my branch (am-CompModel)
2. type start WorkforceManagement.sln (this should open in VS)
3. Open the Computer Model
4. Make sure the get and set information matches data from sql file


if you don't do this you will be
**FIRED**
